Resources/public/Admin.js

```diff
@@ -312,7 +312,7 @@ var Admin = {
         Admin.log('[core|setup_collection_counter] setup collection counter', subject);
 
         // Count and save element of each collection
-        var highestCounterRegexp = new RegExp('_([0-9])+$');
+        var highestCounterRegexp = new RegExp('_([0-9]+)$');
```

Comment:
I think the original `+` should be kept.

For example, `foo_bar_10_email` will not match,